source/api_cc/src/DeepPot.cc

```diff
@@ -218,32 +191,18 @@ init (const std::string & model, const int & gpu_rank, const std::string & file_
   else
     graph_def.ParseFromString(file_content);
   int gpu_num = -1;
-  #if GOOGLE_CUDA
-  cudaGetDeviceCount(&gpu_num); // check current device environment
+  #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
I feel that we could define `GOOGLE_CUDA || TENSORFLOW_USE_ROCM` in another place (for example `USE_DEVICE`), so when we add more devices, we do not need to modify these conditions.

Reply:
> I feel that we could define `GOOGLE_CUDA || TENSORFLOW_USE_ROCM` in another place (for example `USE_DEVICE`), so when we add more devices, we do not need to modify these conditions.

May not be a good idea, because we are supporting more accelerators (device) whose name may not be "gpu"